documentation/rails.md

```diff
@@ -0,0 +1,124 @@
+Rails configuration
+===================
+
+To use this file with Rails you have two choices:
```

Comment:
Reword. Maybe something like:
"Rails can be configured to use the manifest file in one of two ways"